WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyControlTileViewModel.cs

```diff
@@ -20,21 +20,26 @@ public partial class PrivacyControlTileViewModel : TileViewModel, IPrivacyRingPr
 	[AutoNotify] private string _percentText = "";
 	[AutoNotify] private string _balancePrivateBtc = "";
 	[AutoNotify] private bool _hasPrivateBalance;
+	[AutoNotify] private bool _showPrivacyBar;
 
-	public PrivacyControlTileViewModel(WalletViewModel walletVm, IObservable<Unit> balanceChanged)
+	public PrivacyControlTileViewModel(WalletViewModel walletVm, IObservable<Unit> balanceChanged, bool showPrivacyBar = true)
 	{
 		_wallet = walletVm.Wallet;
 		_walletVm = walletVm;
 		_balanceChanged = balanceChanged;
+		_showPrivacyBar = showPrivacyBar;
 
 		ShowDetailsCommand = ReactiveCommand.Create(ShowDetails);
 
-		PrivacyBar = new PrivacyBarViewModel(_walletVm, _balanceChanged);
+		if (_showPrivacyBar)
+		{
+			PrivacyBar = new PrivacyBarViewModel(_walletVm, _balanceChanged);
+		}
 	}
 
 	public ICommand ShowDetailsCommand { get; }
 
-	public PrivacyBarViewModel PrivacyBar { get; }
+	public PrivacyBarViewModel? PrivacyBar { get; }
```

Comment:
I always try to avoid nulls. That's why I would like to have a solution that doesn't accept them :) 

I wonder if it would be nice to make this non-nullable and have a boolean like `IsPrivacyBarVisible` so you can disable in the view according to it?

Reply:
@SuperJMN the `PrivacyBarViewModel` constructor creates a bunch of observables, that I would rather not create if they're not necessary.

Making the property non-nullable makes the compiler cry that it must be non null when exiting `PrivacyControlTileViewModel` ctor. I would rather not do that.